.gitignore

```diff
@@ -117,3 +117,9 @@ gnulib/unistd.h
 gnulib/wchar.h
 gnulib/wctype.h
 gnulib/sys/types.h
+compile_commands.json
+libutil.a
+m6eb
+peg/thrift.c
+peg/thrift.h
+.cache/
```

Comment:
These changes may be nothing to do with improving the support for Basic.
The change itself is acceptable. So could you make a commit for this change or make a pull request for this change?

Reply:
if you want me too ... the problem was that those file got created while using clangd (lsp) + compile for testing on my maching. So i put them in the gitignore so they won't be inside of the commit ;)